.github/dependabot.yml

```diff
@@ -0,0 +1,78 @@
+version: 2
+
+updates:
+  # Update npm packages
+  - package-ecosystem: npm
+    directory: /
+    open-pull-requests-limit: 10
+
+    # Group packages into shared PR
+    groups:
```

Comment:
I suspect some of these packages between lines `10-51` things don't exist in this repo? `webpack`, `sass`, `source-map-loader` and a few otehrs. And there will be ones in the `package.json` of this repo that are not in here